integrated_channels/moodle/exporters/content_metadata.py

```diff
@@ -0,0 +1,77 @@
+"""
+Content metadata exporter for Moodle
+"""
+
+import time
+
+from logging import getLogger
+
+from integrated_channels.integrated_channel.exporters.content_metadata import ContentMetadataExporter
+
+LOGGER = getLogger(__name__)
+
+
+class MoodleContentMetadataExporter(ContentMetadataExporter):
+    """
+        Moodle implementation of ContentMetadataExporter.
+    """
+    DATA_TRANSFORM_MAPPING = {
+        'fullname': 'key',
+        'shortname': 'key',
+        'idnumber': 'number',
+        'summary': 'description',
+        'startdate': 'start',
+        'enddate': 'end',
+        'categoryid': 'categoryid',
+    }
+
+    LONG_STRING_LIMIT = 2000
```

Comment:
given the fact we're passing this description into a query string, we may actually want a shorter limit here or to only use the short description from the catalog. TBD